torch/nn/quantized/_reference/modules/conv.py

```diff
@@ -0,0 +1,136 @@
+        # current quantized conv1d is using Conv2dPackedParams
+        # TODO: we should fix this if we implemenet Conv1dPackedParams
+        self._conv1d_stride = _single(self.stride[0])
+        self._conv1d_padding = _single(self.padding[0])
+        self._conv1d_dilation = _single(self.dilation[0])
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x_dequant = x.dequantize()
+        weight_dequant = self._qweight.dequantize()
+        float_result = F.conv1d(
+            x_dequant, weight_dequant, self._bias, self._conv1d_stride,
+            self._conv1d_padding, self._conv1d_dilation, self.groups)
+        # NEEDFIX: we don't have dtype in the Linear module APIs right now!
+        result = torch.quantize_per_tensor(
+            float_result, self.scale, self.zero_point, torch.quint8)
+        return result
+
+    def _get_name(self):
+        return 'QuantizedConv1d(Reference)'
+
+    @torch.jit.export
+    def __setstate__(self, state):
+        self.in_channels = state[0]
+        self.out_channels = state[1]
+        self.kernel_size = state[2]
+        self.stride = state[3]
+        self.padding = state[4]
+        self.dilation = state[5]
+        self.transposed = state[6]
+        self.output_padding = state[7]
+        self.groups = state[8]
+        self.padding_mode = state[9]
+        self.set_weight_bias(state[10], state[11])
+        self.scale = state[12]
+        self.zero_point = state[13]
+        self.training = state[14]
+        self._conv1d_stride = (self.stride[0],)
+        self._conv1d_padding = (self.padding[0],)
+        self._conv1d_dilation = (self.dilation[0],)
+
+class Conv2d(_ConvNd, nnq.Conv2d):
+    def __init__(self, in_channels, out_channels, kernel_size, stride=1,
+                 padding=0, dilation=1, groups=1, bias=True,
+                 padding_mode='zeros'):
+        nnq.Conv2d.__init__(
+            self, in_channels, out_channels, kernel_size, stride, padding, dilation,
+            groups, bias, padding_mode)
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x_dequant = x.dequantize()
+        weight_dequant = self._qweight.dequantize()
+        float_result = F.conv2d(
+            x_dequant, weight_dequant, self._bias, self.stride,
+            self.padding, self.dilation, self.groups)
+        # NEEDFIX: we don't have dtype in the Linear module APIs right now!
+        result = torch.quantize_per_tensor(
+            float_result, self.scale, self.zero_point, torch.quint8)
+        return result
+
+    def _get_name(self):
+        return 'QuantizedConv2d(Reference)'
+
+class Conv3d(_ConvNd, nnq.Conv3d):
+    def __init__(self, in_channels, out_channels, kernel_size, stride=1,
+                 padding=0, dilation=1, groups=1, bias=True,
+                 padding_mode='zeros'):
+        nnq.Conv3d.__init__(
+            self, in_channels, out_channels, kernel_size, stride, padding, dilation,
+            groups, bias, padding_mode)
```

Comment:
and this?

Reply:
this is multiple inheritance, can we do that?